src/compiler/utilities.ts

```diff
@@ -1177,6 +1177,40 @@ namespace ts {
         return false;
     }
 
+    // Return true if the given identifier is classified as an IdentifierName

```

Comment:
What does it mean to be an IdentifierName and why is it significant? I know what this function is doing in the context of this PR, but for any future reader it may not be clear.